Utils/layouts/index.js

```diff
@@ -66,6 +66,36 @@ const BreadAnchor = styled(Anchor)`
   }
 `;
 
+const MobileBreadAnchor = styled(BreadAnchor)`
+  text-decoration: none;
+  border: none;
+  font-size: 14px;
+  padding: 0;
+  &:before {
+    font-size: 24px;
+    line-height: 0;
+    color: #96b4c0;
+    position: relative;
+    top: 4px;
+    padding: 0;
+    content: "\\2190  ";
+    text-decoration: none;
+  }
+  &:after {
+    content: "";
+  }
+`;
```

Comment:
I think we should add a hover state for this element to match other anchors.

Reply:
It is extending BreadAnchor (which already has the hover state) and overriding some of its style which does not include hover state. Beside, mobile browsers do not have hover state, I believe.

Reply:
Good point about hover state on mobile. My preference is to include it nonetheless since media queries based on viewport width don't test specifically for a mobile device, just the viewport size, so the user may be using a non-mobile device. 
The hover state that is inherited is `text-decoration: none;` which I don't believe is correct for this element, so if we want to apply that here it will need to over-ride that declaration.